.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,103 @@
+          cp .env apps/backend/${{ matrix.service }}/.env
+      - name: Start Docker container for integration tests
+        run: |
+          docker compose -f docker-compose.test.yaml up user-test-db -d
+          for i in {1..30}; do
+            if docker compose -f docker-compose.test.yaml ps | grep "healthy"; then
+              echo "Infrastructures are healthy"
+              break
+            fi  
+            echo "Waiting for infrastructures to be healthy...($i/30)"
+            sleep 10
+          done
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)